firmware/src/mcu/samd11/usb_descriptors.c

```diff
@@ -50,7 +51,7 @@ tusb_desc_device_t const desc_device =
 	// These are a unique VID/PID for development LUNA boards.
 	.idVendor           = 0x1d50,
 	.idProduct          = 0x615c,
-	.bcdDevice          = (_BOARD_REVISION_MAJOR_ << 8) | _BOARD_REVISION_MINOR_,
+	.bcdDevice          = 0,
```

Comment:
If we expose the enum, this could be `BOARD_REV_UNKNOWN` instead of `0`.

Reply:
We would need to define a generic `BOARD_REV_UNKNOWN` in `board_rev.h` just for that. I removed the line entirely to avoid confusion, but let me know if you still think the other option is better.